src/screens/Dashboard.js

```diff
@@ -6,10 +6,12 @@
  */
 
 import React from 'react';
+import ReactLoading from 'react-loading';
 import { connect } from "react-redux";
-import LineChartRealTime from '../components/LineChartRealTime';
```

Comment:
If we're removing the charts, we should do it properly. Maybe we should have a PR just for that, as there are quite a few changes:
1. remove the chart libs from `package.json`;
2. remove files that won't be used anymore, like `components/LineChartRealTime.js`;
3. refactor the redux state. We currently keep an array of the latest data received. I understand we do it for the charts, so there's no need to continue that;

Reply:
I will create an issue to do that in another PR. I guess it's better not to mix those things.